tests/ports/psoc6/hw_ext/sdcard.py

```diff
@@ -12,7 +12,6 @@
 board = os.uname().machine
 if "CY8CPROTO-062-4343W" in board:
     sdcard_config = {
-        "slot": 1,
```

Comment:
umm why? slot is still an arg no? It's removed because there is no test for checking slot and saying it's ignored.?

Reply:
I just removed because it is ignored, but will want, I can also keep it and add to the expect all the warns appearing.